metrics/api/src/main/java/io/helidon/metrics/api/Metrics.java

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.metrics.api;
+
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * A main entry point to the Helidon metrics implementation, allowing access to the global meter registry and providing shortcut
+ * methods to register and locate meters in the global registry and remove meters from it.
+ */
+public interface Metrics {
```

Comment:
Given the general change from "metric" to "meter", I wonder if this shouldn't be named `Meters`.

Reply:
Sigh. I know. 

This is consistent with Micrometer's naming, and there's also the point that some of the methods on this interface are not directly dealing with meters but with other things as well. So the more general name "metrics" (applying to the functional area) seemed OK.